integration/kubernetes/operator/presto/config/samples/_v1alpha1_prestocluster.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: alluxio.com/v1alpha1
+kind: PrestoCluster
+metadata:
+  name: prestocluster-sample
+spec:
+  image: "beinan6666/prestodb"
```

Comment:
As the Kubernetes cluster needs permission to pull images from the repository, is it better to replace it with a public one?